rotkehlchen/chain/ethereum/modules/curve/pools_cache.py

```diff
@@ -0,0 +1,294 @@
+from typing import Set
+
+from eth_utils import to_checksum_address
+
+from rotkehlchen.assets.asset import UnderlyingToken
+from rotkehlchen.assets.utils import get_or_create_evm_token
+from rotkehlchen.chain.ethereum.constants import ZERO_ADDRESS
+from rotkehlchen.chain.ethereum.manager import CURVE_POOLS_MAPPING_TYPE, EthereumManager
+from rotkehlchen.chain.ethereum.types import string_to_evm_address
+from rotkehlchen.chain.ethereum.utils import multicall_specific
+from rotkehlchen.chain.evm.contracts import EvmContract
+from rotkehlchen.constants import ONE
+from rotkehlchen.constants.ethereum import (
+    CURVE_ADDRESS_PROVIDER,
+    CURVE_METAPOOL_FACTORY_ABI,
+    CURVE_REGISTRY_ABI,
+    ETH_SPECIAL_ADDRESS,
+)
+from rotkehlchen.constants.resolver import ChainID
+from rotkehlchen.db.drivers.gevent import DBCursor
+from rotkehlchen.globaldb import GlobalDBHandler
+from rotkehlchen.types import (
+    CURVE_POOL_PROTOCOL,
+    ChecksumEvmAddress,
+    EvmTokenKind,
+    GeneralCacheType,
+)
+from rotkehlchen.utils.misc import hex_or_bytes_to_address
+
+
+def clear_curve_pools_cache(write_cursor: DBCursor) -> None:
+    """Clears all curve pools related cache entries in the globaldb.
+    Doesn't raise anything unless cache entries were inserted incorrectly."""
```

Comment:
If a function does not raise an exception you don't need to write anything.

```suggestion
    """Clears all curve pools related cache entries in the globaldb"""
```